gradle-plugin/src/main/groovy/mozilla/telemetry/glean-gradle-plugin/GleanGradlePlugin.groovy

```diff
@@ -567,6 +567,16 @@ except:
                 }
                 because 'use GeckoView Glean instead of standalone Glean'
             }
+
+            incoming.afterResolve {
```

Comment:
Can you add a changelog entry for this?

Reply:
Yup, change the version a-c depends on to something different from what m-c uses, and BOOM, it breaks. See also [this Fenix PR](https://github.com/mozilla-mobile/fenix/pull/20889) that does the same. If we get this released and into m-c first, then we don't need that in Fenix.